sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -75,7 +77,7 @@ class ServiceBusMessage(object):  # pylint: disable=too-many-public-methods,too-
     """
 
     def __init__(self, body, **kwargs):
-        # type: (Union[str, bytes], Any) -> None
+        # type: (Optional[Union[str, bytes]], Any) -> None
```

Comment:
do we want to make it optional? I know we should allow none, but thought we dont really want to document it.

Reply:
If there was a way to not make MyPy complain from passing None without this type, I'd agree, but I don't want to make life harder for folks who have that and want to pass None because it's not technically incorrect.  Since it's a type signature I don't believe we're tied to backcompat too, if we end up wanting to change this/it causes confusion (as the underlying behavior won't change), but I'll make sure we have the flexibility to reassess this before I merge it.

Reply:
(confirmed that we can be flexible with this)